2017/P_U1/main.cpp

```diff
@@ -0,0 +1,104 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+using namespace std;
+
+struct Spalva
+{
+    string reiksme;
+    int r;
+    int g;
+    int b;
+};
+
+int hexToInt(string hex)
+{
+    int result = 0;
+
+    if (hex[0] - 48 < 10)
+    {
+        result += 16 * (hex[0] - 48);
+    }
+    else
+    {
+        result += 16 * (hex[0] - 55);
+    }
+
+    if (hex[1] - 48 < 10)
+    {
+        result += hex[1] - 48;
+    }
+    else
+    {
+        result += hex[1] - 55;
+    }
```

Comment:
Both conditional statements are the same and can be extracted. Or is there a better way to convert hex strings?